ironfish-cli/src/commands/start.ts

```diff
@@ -201,6 +207,9 @@ export default class Start extends IronfishCommand {
       this.sdk.internal.set('telemetryNodeId', uuid())
       await this.sdk.internal.save()
     }
+    if (enableWallet !== undefined && enableWallet !== this.sdk.config.get('enableWallet')) {
```

Comment:
nit: Can this be moved above the conditional? It's a little easier to see where all the config overrides are happening together.

Reply:
Done